lib/apis/v3/felixconfig.go

```diff
@@ -214,6 +214,8 @@ type FelixConfigurationSpec struct {
 	DebugDisableLogDropping         *bool            `json:"debugDisableLogDropping,omitempty"`
 	DebugSimulateCalcGraphHangAfter *metav1.Duration `json:"debugSimulateCalcGraphHangAfter,omitempty" configv1timescale:"seconds"`
 	DebugSimulateDataplaneHangAfter *metav1.Duration `json:"debugSimulateDataplaneHangAfter,omitempty" configv1timescale:"seconds"`
+
+	IptablesNATOutgoingInterfaceFilter *bool `json:"iptablesNATOutgoingInterfaceFilter,omitempty"`
```

Comment:
As per Felix change, this should be a string.

Also, we should validate this value in `validateFelixConfigSpec` in `validator.go` using same regex as used in Felix (and add UT in `validator_test.go`).

Reply:
ack